app/frontend/src/locales/en/translation.json

```diff
@@ -142,6 +148,8 @@
             "Enables the Azure AI Search semantic ranker, a model that re-ranks search results based on semantic similarity to the user's query.",
         "useQueryRewriting":
             "Enables Azure AI Search query rewriting, a process that modifies the user's query to improve search results. Requires semantic ranker to be enabled.",
+        "reasoningEffort":
```

Comment:
Please translate for other files. (Perhaps we need a script for this..but its not that often yet)